stdlib/multiprocessing/pool.pyi

```diff
@@ -82,7 +82,7 @@ class Pool:
         func: Callable[..., _T],
         args: Iterable[Any] = (),
         kwds: Mapping[str, Any] = {},
-        callback: Callable[[_T], object] | None = None,
+        callback: Callable[[Iterable[_T]], object] | None = None,
```

Comment:
I don't think this is correct. `apply_async`s callback gets passed a single result.

Reply:
Sorry, this was my mistake.